identity-account-storage/src/storage/memstore.rs

```diff
@@ -326,6 +352,20 @@ impl Storage for MemStore {
                 .map_err(Error::DecryptionFailure)?;
             try_decrypt(&derived_secret, encryption_algorithm, &data)
           }
+          CekAlgorithm::ECDH_ES_A256KW(agreement) => {
+            let shared_secret: [u8; 32] = X25519::key_exchange(key_pair.private(), &public_key)?;
+            let derived_secret: Vec<u8> =
+              concat_kdf(cek_algorithm.name(), Aes256Kw::KEY_LENGTH, &shared_secret, agreement)
+                .map_err(Error::DecryptionFailure)?;
+
+            let mut cek: Vec<u8> = vec![0; data.encrypted_cek.len() - Aes256Kw::BLOCK];
```

Comment:
I copied this from the stronghold implementation (which is based on a crypto.rs test), but left out the handling of that panic, because I figured the `MemStore` won't be used in production, and it makes the code more readable. But it's good to be overly cautious, in case anyone copies the implementation.

The crypto.rs test for reference: https://github.com/iotaledger/crypto.rs/blob/1e419b2a9f6299b873d04646c911c79c3537040b/tests/aes_kw.rs#L26. I also checked with someone from crypto.rs on a closely related topic, so I think that should be good.